lib/plugins.js

```diff
@@ -83,13 +83,15 @@ const loadPackages = rootPath => {
 };
 
 const getPlugins = (packages, config) => {
-  return packages
-  .filter(plugin => plugin && plugin.prototype && plugin.prototype.brunchPlugin)
-  .map(Plugin => {
-    const instance = new Plugin(config);
-    instance.brunchPluginName = Plugin.brunchPluginName;
-    return instance;
-  });
+  return (
```

Comment:
I believe `.filter` and `.map` should be indented, without extra braces, they should fail `	
no-extra-parens` rule.